### DISCUSSION TITLE
Hacktoberfest 2021

### DISCUSSION THREAD

**Discussion by jidicula** [pelletier/go-toml#611] [General]
@pelletier will this repo be participating in Hacktoberfest this year?

**Comment by pelletier** [comment]
Sure! I've added go-toml to the [`hacktoberfest`](https://github.com/topics/hacktoberfest) topic. For anyone interested I strongly suggest looking at the [v2 issues](https://github.com/pelletier/go-toml/issues?q=is%3Aissue+is%3Aopen+label%3Av2), and especially this one that has a lot of potentially for small fixes: https://github.com/pelletier/go-toml/issues/613.

⚠️ go-toml work is happening on the `v2` branch. Despite the repository having the `hacktoberfest` topic, it seems like they are not able to accurately track accepted pull requests against that branch as contributions to the event. To make sure your contribution counts, ask me to apply the tag `hacktoberfest-accepted` on your pull request once it is merged.
